ClientApp/viking-vault/src/App.tsx

```diff
@@ -12,7 +12,7 @@ const App: React.FC = () => {
     <div className="App">
       <Router>
         <Route path="/register/" exact component={RegisterForm} />
-        <Route path="/login/" exact component={LoginForm}/>
+        <Route path="/" exact component={LoginForm}/>
```

Comment:
I changed the routes so that the Login page is now on / and not on /login to avoid homepage redirects. If a user comes to the page it shouldn't be redirected to /login